target/linux/kirkwood/patches-4.19/104-ea3500.patch

```diff
@@ -77,20 +77,15 @@
 +		};
 +	};
 +
-+	switches {
-+		#address-cells = <1>;
-+		#size-cells = <0>;
++	mvsw61xx {
```

Comment:
I suspect this should be mvsw61xx@10 as for ea4500?

Reply:
My patch is reverting changes of this node, which was made in #1694.

BTW. Every dts of this node is without address (It is used only in couple devices):
````
grep -rn "mvsw61xx* {" target/linux/ 
target/linux/mvebu/patches-4.14/210-clearfog_switch_node.patch:7:+	mvsw61xx {
target/linux/mvebu/patches-4.14/003-add_switch_nodes.patch:8:+	mvsw61xx {
target/linux/mvebu/patches-4.14/003-add_switch_nodes.patch:27:+	mvsw61xx {
target/linux/mvebu/patches-4.19/210-clearfog_switch_node.patch:7:+	mvsw61xx {
target/linux/mvebu/patches-4.19/003-add_switch_nodes.patch:8:+	mvsw61xx {
target/linux/mvebu/patches-4.19/003-add_switch_nodes.patch:27:+	mvsw61xx {
target/linux/kirkwood/patches-4.14/104-ea3500.patch:80:+	mvsw61xx {
target/linux/kirkwood/patches-4.14/105-ea4500.patch:9:+	mvsw61xx {
````

Reply:
> BTW. Every dts of this node is without address (It is used only in couple devices):

In principle, it should be there (DTS requires the first reg address to be appended with @reg). But I won't care about that now.

> My patch is reverting changes of this node, which was made in #1694.

Ah sorry, I didn't read carefully enough. In this case it's better to not touch the @10 at this time.

BTW: It would be interesting to provide more details on this topic (in the commit message). Unfortunately, neither your revert nor the initial commit is very chatty about that (I am not blaming you here), so at the moment it looks like a "because it works" kind of patch.

Reply:
>In principle, it should be there (DTS requires the first reg address to be appended with @reg). But I won't care about that now.

I know, that is convention. IMO it should be changed in one commit for all affected dtses.

>so at the moment it looks like a "because it works" kind of patch to me.

mv61xx is platform driver. And platform device should be declared in "/" part of dts. If we add another part of tree, driver will be never probed. (Correct me if I'm wrong.)

I will add some info to commit message.